docs/src/pages/system/advanced/advanced.md

```diff
@@ -4,12 +4,15 @@
 
 ## Adding the `sx` prop to your custom components
 
-The `unstable_styleFunctionSx` utility adds the support for the `sx` to your own components. Normally you would use the `Box` components from `@material-ui/core` at the root of your component tree. If you would like to use the system independently from Material-UI, this utility will give you the same capabilities, while having a smaller bundle size.
+The `unstable_styleFunctionSx` utility adds the support for the [`sx` prop](/system/basics/#the-sx-prop) to your own components.
+Normally you would use the `Box` component from `@material-ui/core` at the root of your component tree.
+If you would like to use the system independently from Material-UI, the `unstable_styleFunctionSx` utility will give you the same capabilities, while having a smaller bundle size.
```

Comment:
The "this" refernce was more than one sentence away. I noticed that I had to stop reading to understand what `this` is referring to (I wasn't sure if "this" referred to `Box` or `unstable_styleFunctionSx` even though `unstable_styleFunctionSx` is implied by "this utility".